segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/DurableLog.java

```diff
@@ -293,6 +297,21 @@ public boolean isOffline() {
         return !this.delayedStart.isDone();
     }
 
+    @Override
+   public boolean isInitialized() {
+       try {
+           return this.durableDataLog.loadMetadata() == null;
```

Comment:
This method sounds like a simple getter, but it appears it's actually doing initialization at least partially if it was not done.

Reply:
This is just a getter and does not do initialization.